core/state/statedb.go

```diff
@@ -44,6 +44,14 @@ import (
 
 const defaultNumOfSlots = 100
 
+const (
+	// storageDeleteLimit denotes the highest permissible memory allocation
+	// employed for contract storage deletion.
+	// for now, only hashScheme-based  supported, so no need to delete storages,
+	// pathScheme-based should be supported with `fastDeleteStorage` together.
```

Comment:
Does the `FastDeleteStorage` mentioned here refer to this code section (https://github.com/ethereum/go-ethereum/blob/4e1e37323d87967708b58546f040990db7c2820c/core/state/statedb.go#L951-L993) in the go-ethereum repository?

 If so, I don't think pbss is relevant because FastDeleteStorage only deletes the storage inside the snapshot. It is only executed when snap is enabled.

Reply:
It leverages the associated state snapshot for fast storage iteration
It's just a improvement.

I will modify this PR